site/content/docs/5.2/customize/color.md

```diff
@@ -167,6 +167,13 @@ Check out [our Sass maps and loops docs]({{< docsref "/customize/sass#maps-and-l
 
 ## All colors
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+Some of the colors below do not respect the Orange Design System specifications.
```

Comment:
It might need some detailed explanations here.

Reply:
OK. What about this new proposition?
> Some of the colors below, especially the **variants suffixed by** `-100` **to** `-900`, do not belong the Orange Design System specifications.
> Please refer to our Boosted [Orange's colors]({{< docsref "/customize/color#oranges-colors" >}}) section and to the [Color](https://system.design.orange.com/0c1af118d/p/7059a5-colour/b/17b829) guidelines on the Orange Design System website.

Reply:
Resolved ?